src/config/index.ts

```diff
@@ -1,21 +1,31 @@
 import { PublicKey } from '@solana/web3.js';
-import { Network } from '../models';
-import * as devnet from './devnet';
-import * as mainnet from './mainnet';
-import * as staging from './staging';
+import { ClientEnv, Network } from '../models';
+import * as devnetProd from './devnet/production';
+import * as mainnetProd from './mainnet/production';
+import * as devnetStaging from './devnet/staging';
 import { Vault, VaultProtoConfig, Token } from './types';
 
 export interface Config {
-  defaultProgramId: PublicKey;
+  programId: PublicKey;
   tokens: Record<string, Token>;
   vaultProtoConfigs: Record<string, VaultProtoConfig>;
   vaults: Record<string, Vault>;
 }
 
-export const Configs: Record<Network, Config> = {
-  [Network.MainnetProd]: mainnet,
-  [Network.DevnetProd]: devnet,
-  [Network.DevnetStaging]: staging,
-};
-
+export function getConfig(network: Network, clientEnv: ClientEnv): Config {
+  switch (network) {
+    case Network.Mainnet:
+      return mainnetProd;
```

Comment:
Add a comment here saying that we share the same contract env for both mainnet-prod and mainnet-staging